bin/main

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env ruby
+
+puts "Hello World\n\n"
+
+BOARD_COLS = 3
+BOARD_ROWS = 3
+$Board = []
+$PlayerTurn = nil
+
+class String
+  def gray
+    "\e[38m#{self}\e[0m" 
+  end
+  def red
+     "\e[31m#{self}\e[0m" 
+  end
+  def green
+    "\e[32m#{self}\e[0m"
+  end
+  def blue
+    "\e[34m#{self}\e[0m"
+  end
+end
+
+def initializeBoard
+  for i in 1..BOARD_COLS*BOARD_ROWS
+    $Board << i
+  end
+end
+
+initializeBoard()
+
+def displayBoard
+  for i in 0..(BOARD_COLS*BOARD_ROWS)+((BOARD_ROWS-1)*BOARD_COLS)-1
+      if i < BOARD_COLS
+        print " #{$Board[i]}" unless i == BOARD_COLS-1
```

Comment:
- [ ] The game board should be displayed after each move

Reply:
Hey yeah, we did all of the aforementioned changes already in the Game Logic branch...
We only created the display functions here in the user_interface branch

Reply:
All our other branches have been approved